mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```diff
@@ -73,11 +79,20 @@
                         "type": "integer"
                     },
                     "uniqueItems": true
+                },
+                "special_weights": {
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                },
+                "range_weight": {
+                    "type": "number"
                 }
             },
             "required": ["type", "default", "range"],
             "not": {
-                "required": ["values"]
+                "required": ["values", "weights"]
```

Comment:
What's this about?

Reply:
Ah, I see, `weights` is for `categorical` `values` and `special_weight` and `range_weight` are for "numeric" `ranges` and `specials` 👍

Reply:
For naming, does it make sense to have `values_weights` instead of just `weights` then to be consistent with the other naming conventions?

Reply:
done!